_posts/2024-05-23-geoserver-2-25-1-released.md

```diff
@@ -0,0 +1,121 @@
+
+Task:
+
+* [GEOS-11360](https://osgeo-org.atlassian.net/browse/GEOS-11360) Upgrade Apache POI from 4.1.1 to 5.2.5
+* [GEOS-11362](https://osgeo-org.atlassian.net/browse/GEOS-11362) Upgrade Spring libs from 5.3.32 to 5.3.33
+* [GEOS-11374](https://osgeo-org.atlassian.net/browse/GEOS-11374) Upgrade Spring version from 5.3.33 to 5.3.34
+* [GEOS-11375](https://osgeo-org.atlassian.net/browse/GEOS-11375) GSIP 224 - Individual contributor clarification
+* [GEOS-11388](https://osgeo-org.atlassian.net/browse/GEOS-11388) Update ImageIO-EXT to 1.4.10
+* [GEOS-11393](https://osgeo-org.atlassian.net/browse/GEOS-11393) Upgrade commons-io from 2.12.0 to 2.16.1
+* [GEOS-11395](https://osgeo-org.atlassian.net/browse/GEOS-11395) Upgrade guava from 32.0.0 to 33.2.0
+* [GEOS-11397](https://osgeo-org.atlassian.net/browse/GEOS-11397) App-Schema Includes fix Integration Tests
+* [GEOS-11402](https://osgeo-org.atlassian.net/browse/GEOS-11402) Upgrade PostgreSQL driver from 42.7.2 to 42.7.3
+* [GEOS-11403](https://osgeo-org.atlassian.net/browse/GEOS-11403) Upgrade commons-text from 1.10.0 to 1.12.0
+* [GEOS-11404](https://osgeo-org.atlassian.net/browse/GEOS-11404) Upgrade commons-codec from 1.15 to 1.17.0
+
+For the complete list see [2.25.1](https://github.com/geoserver/geoserver/releases/tag/2.25.1) release notes. 
+
+## Community Updates
+
+Community module development:
+
+* [GEOS-11040](https://osgeo-org.atlassian.net/browse/GEOS-11040) Could not get a ServiceInfo for service Features thus could not check if the service is enabled
+* [GEOS-11330](https://osgeo-org.atlassian.net/browse/GEOS-11330) OAuth2 kid verification should be optional
+* [GEOS-11339](https://osgeo-org.atlassian.net/browse/GEOS-11339) Introducing the Features Autopopulate Community Plugin
+* [GEOS-11340](https://osgeo-org.atlassian.net/browse/GEOS-11340)  WFS Freemarker HTML Outputformat
+* [GEOS-11345](https://osgeo-org.atlassian.net/browse/GEOS-11345) STAC Conformance URIs need to be updated to v1.0.0
+* [GEOS-11348](https://osgeo-org.atlassian.net/browse/GEOS-11348) JMS cluster does not allow to publish style via REST "2 step" approach
+* [GEOS-11358](https://osgeo-org.atlassian.net/browse/GEOS-11358) Feature-Autopopulate Update operation does not apply the Update Element filter
+* [GEOS-11381](https://osgeo-org.atlassian.net/browse/GEOS-11381) Error in OIDC plugin in combination with RoleService
+* [GEOS-11394](https://osgeo-org.atlassian.net/browse/GEOS-11394) OGC APIs cannot handle time extent when the source data type is java.sql.Date
+
+Community modules are shared as source code to encourage collaboration. If a topic being explored is of interest to you, please contact the module developer to offer assistance. 
+
+# About GeoServer 2.25 Series
+
+Additional information on GeoServer 2.25 series:
+
+* [GeoServer 2.25 User Manual](https://docs.geoserver.org/2.25.x/en/user/)
+* [GeoServer 2024 Roadmap Plannings]({% post_url 2024-01-03-roadmap %}) 
```

Comment:
This hyperlink looks like it needs fixing

Reply:
that is a macro that gets filled in if you run locally